src/routes/app/upload/+layout@.svelte

```diff
@@ -0,0 +1,20 @@
+<script lang="ts">
+  import "@/style/kit.css";
+
+  import MainLayout from "@/lib/components/MainLayout.svelte";
+  import SignedIn from "@/lib/components/common/SignedIn.svelte";
+
+  import Documents from "../sidebar/Documents.svelte";
+  import Projects from "../sidebar/Projects.svelte";
+</script>
+
+<MainLayout>
+  <svelte:fragment slot="navigation">
+    <Documents />
+    <SignedIn>
+      <Projects />
+    </SignedIn>
+  </svelte:fragment>
+
+  <slot slot="content" />
+</MainLayout>
```

Comment:
What does the `@` do in the filename?

Reply:
It resets the layout: https://kit.svelte.dev/docs/advanced-routing#advanced-layouts-layout

Had to clear the layout stack so I could clear the action slot. There may be a better way to do this, without needed a layout component for uploads. We should revisit, especially if we're considering changing up the URL structure.